Header header
string child_frame_id
string intersection_name
int32 intersection_enum
geometry_msgs/Pose pose
string reached
